ds-prep-python-loops/main.py

```diff
@@ -0,0 +1,68 @@
+# Loop Tuples
+record = (1, "Grimdiana", "Bones", "boulders")
+
+row = ''
+
+for x in record:
+  print(x)
+  row = row + str(x) + ", "
+
+print(row)
+
+# Loop Lists
+values_list = [1, 1, 2, 3, 5, 8, 13, 21, 34]
+
+for x in values_list:
+  print(x)
+
+index_list = []
+
+for i in range(len(values_list)):
+  print(i)
+  index_list.insert(0, i)
+
+print(index_list)
+
+for i in index_list:
+  if i % 2 != 0:
+    values_list.pop(i)
+
+print(values_list)
+
+# Loop Sets
+vowels = {"A", "E", "I", "O", "U"}
+parts_of_the_big_letter = {"L", "M", "N", "O", "P"}
+
+for i in vowels:
+  parts_of_the_big_letter.discard(i)
+
+print(parts_of_the_big_letter)
+
+# Loop dictionary
+player_positions = {
+    "Who": "1B",
+    "What": "2B",
+    "I Don't Know": "3B",
+    "Why": "LF",
+    "Because": "CF",
+    "Tomorrow": "P",
+    "Today": "C",
+    "I Don't Care": "SS"
+    }
+
+players = []
+
+for x in player_positions:
+  players.insert(0, x)
+
+print(players)
+
+positions = []
```

Comment:
A bit of a nitpick, but `positions` should be a `set`, not a `list`:

---

![image](https://user-images.githubusercontent.com/16143771/165416457-02fee38c-74b5-4eb6-ba89-6be1ca248a76.png)

Reply:
Thanks! Noted.